src/ui/zcl_abapgit_popups.clas.abap

```diff
@@ -327,6 +339,32 @@ CLASS zcl_abapgit_popups IMPLEMENTATION.
   ENDMETHOD.
 
 
+  METHOD set_starting_position.
+
+    CONSTANTS:
+      lc_min_size TYPE i VALUE 10,
+      lc_min_pos  TYPE i VALUE 5.
+
+    " Magic math to approximate starting position of popup
```

Comment:
I assume this is similar to the logic in cl_bus_abstract_main_screen=>compute_center. If so: the sy fields don't really work if called from within a popup, just fyi.

Reply:
It's similar. The sy fields are evaluated before the popup is called. It works well for the case we have in abapGit